### DISCUSSION TITLE
Asked to fork openupm when attempting to submit my package

### DISCUSSION THREAD

**Discussion by improck** [openupm/openupm#4562] [Q&A] [answered]
After languishing on the Asset Store for years, I've recently updated my editor extension and published it on Github as a package. I'm attempting to submit it to openupm using the [Submitting an Open Source UPM Package](https://openupm.com/packages/add/) page. The instructions at the top say to click the Commit Changes button after submitting. But I don't have that. It's telling me that I need to fork the repo to propose changes. Is something broken or are the instructions just out-of-date?

[Submission link](https://github.com/openupm/openupm/new/master/data/packages/?filename=com.improck.jumpto.yml&message=chore%28data%29%3A%20new%20package%20com.improck.jumpto&value=name%3A%20com.improck.jumpto%0AdisplayName%3A%20Imp%20Rock%20-%20JumpTo%0Adescription%3A%20A%20favorites%20tool%20for%20Unity%0ArepoUrl%3A%20https%3A%2F%2Fgithub.com%2Fimprock%2Fjumpto%0AparentRepoUrl%3A%20null%0AlicenseSpdxId%3A%20%27%27%0AlicenseName%3A%20Custom%20BSD-3-clause%0Aimage%3A%20%27%27%0Atopics%3A%0A%20%20-%20editor-enhancement%0A%20%20-%20utilities%0Ahunter%3A%20improck%0AgitTagPrefix%3A%20%27%27%0AgitTagIgnore%3A%20%27%27%0AminVersion%3A%202.1.0%0Areadme%3A%20upm%3AREADME.md%0AcreatedAt%3A%201705185056088%0A)

Thanks

**Comment by favoyang** [comment, marked as answer]
HI @improck,

Thanks for publishing on the OpenUPM.

![image](https://github.com/openupm/openupm/assets/125390/7678464c-f6b5-4e2e-89bc-09cd7de77c0c)

Please click the "Commit changes" button on the top right to send us a PR.

For the first-time contributor, you need to "fork the repo", because that is the way to send a PR on the GitHub.

Please let me know if you need more help, or perhaps leave a screenshot for troubleshooting.

**Reply by improck** [reply]
> For the first-time contributor, you need to "fork the repo", because that is the way to send a PR on the GitHub.

That was the part I was missing. I'm new to Github. You may want to consider adding a note about that on the instructions page for others that are not familiar with Git/Github.

I've forked and committed the change and it is running a task pending approval. Thank you very much!

**Reply by favoyang** [reply]
Thank you for the suggestion.
